mangaki/mangaki/templates/mangaki/work_detail.html

```diff
@@ -139,6 +139,10 @@
 	(cc) <a href="http://manga-news.com/" target="_blank">Manga-News</a>
       {% endif %}
     </div>
+{% elif object.ext_synopsis %}
```

Comment:
I'm fluent in English, so it doesn't bother me personally that much -- but chunks of text randomly in a foreign language is something that usually gives me a very negative opinion of a website, so I'd like for it not to happen on Mangaki.

Would you be OK with removing this for now and discuss the matter at the next meeting? As you've shown, it'd be only a couple lines to add back. I'm fine with the other way around (adding it, then discussing it and potentially removing it), but I think there is value in limiting the visual changes introduced in this PR.